ocs_ci/ocs/constants.py

```diff
@@ -846,6 +846,7 @@
 MDR_BACKUP_SCHEDULE_YAML = os.path.join(
     TEMPLATE_MULTICLUSTER_DIR, "backupschedule.yaml"
 )
+MDR_RESTORE_YAML = os.path.join(TEMPLATE_MULTICLUSTER_DIR, "restore.yaml")
```

Comment:
This restore yaml is common for both RDR and MDR, hence a generic name would make it flexible for re-using.

Reply:
changed to DR_RESTORE_YAML